libpod/oci.go

```diff
@@ -210,6 +211,14 @@ func (r *OCIRuntime) createContainer(ctr *Container, cgroupParent string) (err e
 	// 0, 1 and 2 are stdin, stdout and stderr
 	cmd.Env = append(r.conmonEnv, fmt.Sprintf("_OCI_SYNCPIPE=%d", 3))
 	cmd.Env = append(cmd.Env, fmt.Sprintf("_OCI_STARTPIPE=%d", 4))
+	if notify, ok := os.LookupEnv("NOTIFY_SOCKET"); ok {
```

Comment:
Do we want to make this conditional at all, or is it OK if libpod does this in CRI-O as well?